server/src/infra/repositories/person.repository.ts

```diff
@@ -118,4 +126,13 @@ export class PersonRepository implements IPersonRepository {
   async getFaceById({ personId, assetId }: AssetFaceId): Promise<AssetFaceEntity | null> {
     return this.assetFaceRepository.findOneBy({ assetId, personId });
   }
+
+  async getRandomFace(personId: string): Promise<AssetFaceEntity | null> {
+    return this.assetFaceRepository
+      .createQueryBuilder('assetFace')
+      .where('assetFace.personId = :personId', { personId: personId })
+      .orderBy('random()')
```

Comment:
Does this actually need to be random? Can we just grab the first face? It looks like using `random()` forces postgres to do a full table scan.

Reply:
This whole thing can probably be rewritten as just 


```typescript
return this.assetFaceRepository.findByOne( { personId });
```

Reply:
This whole thing can probably be rewritten as just 


```typescript
return this.assetFaceRepository.findByOne({ personId });
```